### ISSUE TITLE
Suggestion for unify parameter name in nn.Conv and nn.Linear

### ISSUE THREAD

**Issue by zeyu-hello** [pytorch/pytorch#37342]
The `self.in_channels` in [nn.Conv](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/conv.py#L34) and `self.in_features` in [nn.Linear](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/linear.py#L70) are representing the same kind of param, so I suggest to use same name, for example `self.in_channels`. This is also do for other same situation.

Thanks!

cc @ezyang @SsnL @gchanan

**Comment by lufol** [comment]
I’m not sure this would improve the Conv. 
As the in_channels of the Conv correspond to the number of channels of the input tensor. So for example for an RGB input image this would be 3 input channels (1 channel per color channel). 
On the other hand the in_features of the linear layer actually correspond to the number of features as input. So for the same image as above and reshaped into a 1-dim vector this would be the overall number of pixels in the vector.

**Comment by mrshenli** [comment]
+1 to @lufol's comments above. 

I am adding this to triage meeting to get feedback from the team.

**Comment by ezyang** [comment]
While the appeal for consistency is attractive, it's certainly not worth breaking BC for this. So whatever you'd plan to do, you'd also have to have a plan for making the change backwards compatible.

**Comment by gchanan** [comment]
duplicate of https://github.com/pytorch/pytorch/issues/984.
